app/src/main/res/values/strings.xml

```diff
@@ -1,7 +1,7 @@
 <resources>
     <string name="app_name">Peakality</string>
     <string name="get_score">Get Score!</string>
-    <string name="front_text">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</string>
+    <string name="front_text">Peakality is the greatest app! Show us that you\'re the realest. Find yourself in extreme situations. Get the highest score and prove your friends that you\'re the realest among them.</string>
```

Comment:
Ich würde das erste "Show us that you're the realest." weglassen, da es in etwa das selbe aussagt wie das zweite "prove your friends that you\'re the realest among them". Sonst finde ich den Text sehr gut.

Reply:
Fixed